docs/source/releases/1.17.1/1.17.1-bug-fixes.yaml

```diff
@@ -0,0 +1,164 @@
+bug fixes:
+- title: Bug fixes on 1.17.1
+  description: |
+    Rearranged env files so they start with X.Y.Z, then env tag (e.g., baseinstall, sitetestfull, etc)
+
+    * geoips/environments all files
+
+    Add 20210929 DMW datasets to test_data_abi so test scripts run (L1b required for config-based).
+
+    * abi.static.dmw.imagery_windbarbs_high.sh
+    * abi.config_based_dmw_overlay.sh
+
+    Tiny sector fixes in tests/output and tests/scripts
+
+    * Updated abi dust rgb tiny sector test output.
+      3km -> 10km, new location/coverage ?.
+    * Removed some old config-based Visible tiny sector outputs
+    * Renamed config-based non-dataset-based tiny sector from goes16 to goeseast
+
+      * Final tiny sector is test_goeseast_eqc_3km_landocean
+
+    Use metadata_default vs metadata_tc
+
+    * metadata_default should never change, so only test metadata_tc once, so we only
+      have one integration test output to update when metadata_tc output formatter
+      changes
+    * amsr2.config_based_overlay_output.sh
+
+      * amsr2.tc_overlay.37pct.imagery_annotated_over_Visible/20200518_073601_IO012020_amsr2_gcom-w1_37pct_140kts_95p89_res1p0-cr100-bgVisible.png.yaml
+      * amsr2.tc_overlay.89pct.imagery_annotated_over_Visible/20200518_073601_IO012020_amsr2_gcom-w1_89pct_140kts_98p32_res1p0-cr100-bgVisible.png.yaml
+    * amsr2.config_based_overlay_output_low_memory.sh
+
+      * amsr2.tc_overlay.37pct.imagery_annotated_over_Infrared-Gray/20200518_073601_IO012020_amsr2_gcom-w1_37pct_140kts_95p89_res1p0-cr100-bgInfrared-Gray.png.yaml
+      * amsr2.tc_overlay.89pct.imagery_annotated_over_Infrared-Gray/20200518_073601_IO012020_amsr2_gcom-w1_89pct_140kts_98p32_res1p0-cr100-bgInfrared-Gray.png.yaml
+
+    Partial caching fix for abi_netcdf
+
+    * geoips/plugins/modules/readers/abi_netcdf.py
+
+    Rearrange proj4 strings
+
+    * abi.config_based_output_low_memory.sh and abi.config_based_output.sh
+
+      * abi.tc.Infrared.imagery_clone.low_memory
+      * abi.tc.IR-BD.imagery_clone.low_memory
+    * abi.config_based_output.sh
+
+      * abi.tc.Visible.imagery_clean
+      * abi.tc.IR-BD.imagery_clean
+    * hy2.tc.windspeed.imagery_annotated
+    * ssmi.tc.37pct.imagery_clean
+
+    Environment and minor testing updates
+
+    * Quote additional_args in interactive_pytest.sh so it works with more than one word.
+    * setup/config_geoips - make GEOIPS_LOGGING_LEVEL a supported overrideable env var.
+      and default to info. Also make GEOIPS_PERSISTENT_DATA_DIRS overrideable env var
+      with default to $GEOIPS BASEDIR/../persistent_data_dirs.
+    * output_checkers.py - print out all the files in the output comparison directory when
```

Comment:
This is the update that had broken new test scripts (tried to listdir a non existent directory before creating it.). That bug has since been fixed